python/triqs/gf/tools.py

```diff
@@ -259,8 +259,94 @@ def fit_legendre(g_t, order=10):
     l = np.arange(len(lmesh))
     scale = np.sqrt(2.*l + 1) / mesh.beta
     scale = scale.reshape([len(lmesh)] + [1]*len(g_t.target_shape))
-    
+
     g_l = Gf(mesh=lmesh, target_shape=g_t.target_shape)
     g_l.data[:] = c_l.reshape(g_l.data.shape) / scale
 
     return g_l
+
+
+def discretize_bath(delta_inp, Nb , bandwidth = 3, t_val = 0.0, tol=1e-5):
```

Comment:
Use delta_iw instead of delta_imp throughout?
```suggestion
def discretize_bath(delta_iw, Nb, bandwidth = 3, t_init = 0.0, tol=1e-5):
```